src/components/Superlative.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import axios from 'axios';
+
+export default class Superlative extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      superlativeData: {},
+      studentList: []
+    }
+
+    // axios request to get a list of the students
+    axios.get('/getAllStudents')
+      .then((response) => {
+        this.setState({
+          'studentList': response.data.map((student, index, collection) => {
+            return student.name;
+          })
+        });
+      })
+      .then(() => {
+        console.log('students: ', JSON.stringify(this.state.studentList, undefined, 2));
+      });
+  }
+
+  componentDidMount() {
+    this.getSuperlativeData();
+  }
+
+  getSuperlativeData() {
+    // Example URL: http://localhost:3000/superlative/Most%20Likely%20To%20Be%20An%20Axe%20Murder
+
+    // get the superlative name and undo the url encoding
+    const superlativeName = decodeURIComponent(window.location.href.split('/superlative/')[1]);
+
+    axios.post('/getParticularSuperlative', {
+      'superlativeInfo': {
+        'superlative': superlativeName
+      }
+    })
+      .then((response) => {
+        if (response.data[0]) {
+          this.setState({ superlativeData: response.data[0] });
+        }
+      })
+      .catch((error) => {
+        console.log('getSuperlativeData error', error);
+      });
+  }
+
+  handleVote(event) {
+    event.preventDefault();
+
+    // handle the vote here
+    const person = document.getElementById('input').value;
+    console.log(JSON.stringify(person, undefined, 2));
+
+    if (this.state.studentList.indexOf(person) > -1) {
+      console.log('It is a valid name, so increment the vote count');
```

Comment:
Is this method adding votes to the db yet?

Reply:
Nope, but I figure that I want to get a pull request in now and then another one later instead of a gigantic one, like i did at the start of the project